### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#217] [Esercizi - Bertini] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/249eeb3a-784a-4360-a653-fcaaa6a9d946)

**Comment by cornflexxx** [comment, marked as answer]
## Soluzione analoga a quella in classe
$1)$ Chiamiamo $X_{\Sigma}=X_1+...+X_k$

Osserviamo che $X_i$ è una  variabile aleatoria con distribuzione $G(p)-1$ , oppure per facilitarci i ragionamenti è una variabile aleatoria che conta il numero di fallimenti prima di un successo , infatti ,al contrario di una normale geometrica, il caso del successo non viene contato. Supponiamo che $X_{\Sigma}=N$(dove $N$ è la somma di tutti i fallimenti per ogni $X_i$) allora, questa $v.a$ conterà il numero di fallimenti prima di $k$ successi. Posto in questo modo è un problema che abbiamo già risolto, come per $a$ teste prima di $b$ croci la distribuzione di $X_{\Sigma}$ sarà:

$$\mathbb{P}(X_{\Sigma}=N)=\binom{N+k-1}{k-1} p^k(1-p)^N$$

$2)$ Calcolo la congiunta condizionata alla somma e semplifico i termini :

$$\mathbb{P}(X_1=n_1,...,X_k=n_k|X_1+...+X_k=n)=\frac{\mathbb{P}(X_1=n_1,...,X_k=n_k)}{\mathbb{P}(X_{\Sigma}=n)}=\frac{1}{\binom{n+k-1}{k-1}}$$

$3)$ Risulta proprio che la distribuzione congiunta delle $v.a$, $X_1,...,X_k$ è uniforme  sull'insieme $\Omega$, che rappresenta lo spazio campionario: $\Omega=\\{ (n_1,...,n_k)|n_1+...+n_k=n\\}$

Possiamo ricondurre anche questo problema alle disposizione di n palline in k scatole distinte , la $k-upla$ di $\Omega$ rappresenta proprio la divisione delle n palline , allora risulta abbastanza intuitivo il fatto che ci sia un solo caso in cui ad $X_1$ sono associate esattamente $n_1$ palline e cosi via per tutti gli altri fino a $k$ . Il denominatore è giustificato dal fatto che rappresenta  tutti i casi possibili, quindi una distribuzione $\frac{\text{casi favorevoli}}{\text{casi possibili}}$ è chiaramente uniforme.
